manim/scene/scene_file_writer.py

```diff
@@ -351,6 +351,8 @@ def open_movie_pipe(self, file_path=None):
         self.partial_movie_file_path = file_path
 
         fps = config["frame_rate"]
+        if fps == int(fps):  # fps is integer
+            fps = int(fps)
```

Comment:
I think you can simply do
```suggestion
        fps = int(config["frame_rate"])
```

Reply:
I can't. This is checking whether the float ends in `.0`. If it does, then it converts its to integer... but right below this bit of code it is used for ffmpeg -- and fps is NOT always int.